nomt/src/beatree/ops/update/mod.rs

```diff
@@ -19,6 +19,9 @@ use super::BranchId;
 use branch::{BaseBranch, BranchUpdater, DigestResult as BranchDigestResult};
 use leaf::{BaseLeaf, DigestResult as LeafDigestResult, LeafUpdater};
 
+#[cfg(feature = "benchmarks")]
+pub use leaf::separate;
```

Comment:
maybe find another way to do this. exporting this all from the global namespace is a pretty big code smell.

Reply:
It doesn't seem to exist a perfect solution. The main problem is that there are private functions inside private modules that need to be benchmarked by something outside of the crate. The only drastic solution I see is to start using `#[cfg(bench)]`, which is still a nightly feature.

I hope that what is proposed in the new update of the PR is ok for you!